### PR TITLE
[BOUNTY] borgs no longer start with a cruciform.

### PR DESCRIPTION
**Pull request by Evankhell561**
## Changelog
:cl:
fix: borgs no longer spawn with a cruciform..
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
based

**Review by MadmanMartian — APPROVED** [review]

**Comment by MadmanMartian** [comment]
wait

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1201,7 +1201,7 @@ var/list/rank_prefix = list(\
 				new organ_type(src)
 
 		var/datum/category_item/setup_option/core_implant/I = Pref.get_option("Core implant")
-		if(I.implant_type)
+		if(I.implant_type && (!mind || mind.assigned_role != "Robot"))
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/human.dm:1204]: Might want to check for AIs, too.

Comment (Evankhell561) [reply code/modules/mob/living/carbon/human/human.dm:1204]: the code to create a body for a new player is a mess and as far as I could test the AIs do not suffer from this problem.

File: code/controllers/subsystems/jobs.dm [hunk 2]
```diff
@@ -414,7 +414,7 @@ SUBSYSTEM_DEF(job)
 	BITSET(H.hud_updateflag, SPECIALROLE_HUD)
 	return H
 
-proc/EquipCustomLoadout(var/mob/living/carbon/human/H, var/datum/job/job)
+/proc/EquipCustomLoadout(var/mob/living/carbon/human/H, var/datum/job/job)
```
Comment (Hopekz) [inline code/controllers/subsystems/jobs.dm:417]: crazy

**Review by Hopekz — APPROVED** [review]

**Pull request merged**